src/encoded/types/experiment.py

```diff
@@ -628,9 +635,11 @@ def experiment_summary(self, request, experiment_type, biosample, targeted_facto
         sum_str = request.embed(experiment_type, '@@object')['display_title']
 
         if targeted_factor:
-            target_props = request.embed(targeted_factor, '@@object')
-            target_summary = target_props['display_title']
-            sum_str += ('against ' + target_summary)
+            tstring = ''
+            for tf in targeted_factor:
+                target_props = request.embed(tf, '@@object')
+                tstring += ', {}'.format(target_props['display_title'])
+            sum_str += ('against ' + tstring[2:])
```

Comment:
might need to add a space (e.g. `' against '`) to avoid display titles like `CUT&RUNagainst YFG protein on GM12878`